doc/man3/BIO_sendmmsg.pod

```diff
@@ -0,0 +1,167 @@
+set to the size of the buffer in bytes. If the given B<BIO_MSG> is processed (in
+other words, if the integer returned by the function is greater than or equal to
+that B<BIO_MSG>'s array index), I<data_len> will be modified to specify the
+actual amount of data sent or received.
+
+The I<flags> field of a B<BIO_MSG> provides input per-message flags to the
+invocation. If the invocation processes that B<BIO_MSG>, the I<flags> field is
+written with output per-message flags, or zero if no such flags are applicable.
+
+Currently, no input or output per-message flags are defined and this field
+should be set to zero before calling BIO_sendmmsg() or BIO_recvmmsg().
+
+The I<flags> argument to BIO_sendmmsg() and BIO_recvmmsg() provides global
+flags which affect the entire invocation. No global flags are currently
+defined and this argument should be set to zero.
+
+When these functions are used to send and receive datagrams, the I<peer> field
+of a B<BIO_MSG> allows the destination address of sent datagrams to be specified
+on a per-datagram basis, and the source address of received datagrams to be
+determined. The I<peer> field should be set to point to a B<BIO_ADDR>, which
+will be read by BIO_sendmmsg() and used as the destination address for sent
+datagrams, and written by BIO_recvmmsg() with the source address of received
+datagrams.
+
+Similarly, the I<local> field of a B<BIO_MSG> allows the source address of sent
+datagrams to be specified on a per-datagram basiis, and the destination address
+of received datagrams to be determined. Unlike I<peer>, support for I<local>
+must be explicitly enabled on a B<BIO> before it can be used; see
+BIO_dgram_set_local_addr_enable(). If I<local> is non-NULL in a B<BIO_MSG> and
+support for I<local> has not been enabled, processing of that B<BIO_MSG> fails.
+
+I<peer> and I<local> should be set to NULL if they are not required. Support for
+I<local> may not be available on all platforms; on these platforms, these
+functions always fail if I<local> is non-NULL.
+
+On Windows, if I<local> is specified, the B<BIO_ADDR> it points to will be
+zeroed (address family set to zero) if a local address was not reported by the
+operating system. This occurs when a packet is received which was sent by the
+local system. This occurs regardless of whether the packet's destination address
+was the loopback address or the IP address of a local non-loopback interface.
+This is an OS-specific limitation.
+
+The I<stride> argument must be set to C<sizeof(BIO_MSG)>. This argument
+facilitates backwards compatibility if fields are added to B<BIO_MSG>. Callers
+must zero-initialize B<BIO_MSG>.
+
+I<num_msg> should be sent to the maximum number of messages to send or receive,
+which is also the length of the array pointed to by I<msg>.
+
+Unlike most BIO functions, these functions explicitly support multi-threaded
+use. Multiple concurrent writers and multiple concurrent readers are permitted
```

Comment:
perhaps modify to: "Multiple concurrent writers and multiple concurrent readers of the same BIO are permitted..."